app/views/guests/show.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1><%= @guest.name %>'s Stay at <%= @guest_rooms.hotel.name %></h1>
+<% @guest.rooms.each do |room| %>
+  <p>Guest's Room: <%= room.suite %></p>
+  <p>Nightly Rate: <%= room.rate %></p>
+<% end %>
+<p>Number of Nights: <%= @guest.nights %></p>
```

Comment:
Following up on one of my first comments... since we want to display each room's hotel name, we'd want to display that during the each block: 

<h1><%= @guest.name %>'s Stays </h1>
<% @guest.rooms.each do |room| %>
  <p>Guest's Room: <%= room.suite %></p>
  <p>Nightly Rate: <%= room.rate %></p>
  <p>Hotel Name: <%= room.hotel.name %></p>
<% end %>
<p>Number of Nights: <%= @guest.nights %></p>